cg/cli/workflow/mip_rna/store.py

```diff
@@ -61,7 +57,7 @@ def _gather_files_and_bundle_in_housekeeper(
     try:
         results = hk_api.add_bundle(bundle_data)
         if results is None:
-            print(click.style("analysis version already added", fg="yellow"))
+            click.echo(click.style("analysis version already added", fg="yellow"))
```

Comment:
Why not LOG?

Reply:
since it was styled and print does not support that. It will end up in the log?

Reply:
yes, but click.echo() will write to stdout right I think. Is this what we want?

Reply:
why not `LOG.info("analysis version already added")`. Maybe it's ok that the txt isn't yellow 😄

Reply:
will it be "lost" then somehow?

Reply:
Depends what we do with the logs from this. For all practical purposes I don't think it matters, but everywhere else we use `logging` which I think writes to stderr